### PR TITLE
Intro to Github assignment

### PR DESCRIPTION
**Pull request by kjb589**
Merged branch with code review into my branch. Then, merged into the main branch of my forked repository. Created new commit in which I added the suggestions from the code review.

### CODE REVIEW

File: The Descent-bab978/Program.c [hunk 1]
```diff
@@ -1,52 +1,80 @@
-/* # Descent into CodeChat */
-/* ## Code explanation */
-/*  The *while(1)* loop is what runs the game.
-    Each iterative run of the loop is a turn in the game
-    in which there will be an *input* of the mountains and the *output* to fire at a mountains.
-    The inputs are updated per run of the loop */
+/* # Descent into CodeChat
```
Comment (bjones1) [inline The Descent-bab978/Program.c:1]: Delete this from your PR -- all I want is the modifications to your code.

File: otherother 2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This file is more important than my life
```
Comment (bjones1) [inline otherother 2.txt:1]: Delete all other files not related to this PR.

File: Temperatures_kjb589/kjb589_Temperatures.py [hunk 3]
```diff
@@ -9,20 +9,25 @@
 # considered closest to zero (for instance, if the temperatures are -5 and 5,
 # then display 5).
 #
+# <mark >BAB:</mark> Good explanation I can't find anything to critize here.
```
Comment (bjones1) [inline Temperatures_kjb589/kjb589_Temperatures.py:12]: Remove review comments; instead, integrate them into your code or explain why you didn't in a PR.

**Review by bjones1 — CHANGES_REQUESTED** [review]

File: The Descent-bab978/Program.c (hunk 1 above)
Comment (kjb589) [reply The Descent-bab978/Program.c]: Deleted.

File: otherother 2.txt (hunk 2 above)
Comment (kjb589) [reply otherother 2.txt]: Deleted all files except the folder containing my codingame solution.

File: Temperatures_kjb589/kjb589_Temperatures.py (hunk 3 above)
Comment (kjb589) [reply Temperatures_kjb589/kjb589_Temperatures.py]: Integrated variable name changes, edited some of my own comments to fix grammar, and removed peer review comments.

**Comment by kjb589** [comment]
Please review the changes I've made and request changes if needed.

**Review by bjones1 — CHANGES_REQUESTED** [review]
Would you back out the massive file deletions? I want to keep other student solutions in the repo.

**Comment by kjb589** [comment]
Re-added other students code and reviews. Let me know if there are any other issues.

**Pull request closed without merging**